app/components/global-navigation/search.js

```diff
@@ -34,6 +35,7 @@ export default Component.extend({
 	},
 	/* eslint-disable ember/avoid-leaking-state-in-ember-objects */
 	isEmptyQuery: empty('state.query'),
+	searchId: null,
```

Comment:
in suggestions events there is no such thing like `searchId`, you should name it `suggestionId `